pkg/scheduler/partition.go

```diff
@@ -567,6 +569,9 @@ func (pc *PartitionContext) AddNode(node *objects.Node, existingAllocations []*o
 				metrics.GetSchedulerMetrics().IncFailedNodes()
 				return err
 			}
+			if alloc.IsPlaceholder() {
+				pc.incPhAllocationCount()
+			}
```

Comment:
This one needs to be reversed also if the node is removed.
The node removal path is not currently covered: see line 698 (after change) or 693 (before change) need a decrement placeholder count in that check

Reply:
Now we add and remove in two methods:
* `removeNodeAllocations()` / `removeAllocation()` (one/ all when uuid = "")
* `allocate()` (normal scheduling) / `addAllocation()` (startup)